types/relay-test-utils/lib/unwrapContainer.d.ts

```diff
@@ -4,6 +4,8 @@ import { RelayProp, RelayPaginationProp, RelayRefetchProp } from 'react-relay';
 /**
  * Returns original component class wrapped by e.g. createFragmentContainer
  */
-export function unwrapContainer<Props>(
+declare function unwrapContainer<Props>(
     ComponentClass: ComponentType<Props | RelayProp | RelayPaginationProp | RelayRefetchProp>,
 ): ComponentType<Props>;
+
+export = unwrapContainer;
```

Comment:
I don't remember seeing this `export = ` syntax before. Is it like `modules.exports = `? or `export unwrapContainer`?

Reply:
yeah no clue, but think `export default != module.exports`, and has something to do with esModuleInteropt. 

cc @levibuzolic

Reply:
Yeah I think this looks correct. :+1:

https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require